ksql-common/src/main/java/io/confluent/ksql/util/KsqlConfig.java

```diff
@@ -167,11 +167,19 @@
       "Config to enable or disable transient pull queries on a specific KSQL server.";
   public static final boolean KSQL_QUERY_PULL_ENABLE_DEFAULT = true;
 
-  public static final String KSQL_QUERY_PULL_ROUTING_TIMEOUT_MS_CONFIG =
-      "ksql.query.pull.routing.timeout.ms";
-  public static final Long KSQL_QUERY_PULL_ROUTING_TIMEOUT_MS_DEFAULT = 30000L;
-  public static final String KSQL_QUERY_PULL_ROUTING_TIMEOUT_MS_DOC = "Timeout in milliseconds "
-      + "when waiting for the lookup of the owner of a row key";
+  public static final String KSQL_QUERY_PULL_ENABLE_STANDBY_READS =
+        "ksql.query.pull.enable.stale.reads";
+  private static final String KSQL_QUERY_PULL_ENABLE_STANDBY_READS_DOC =
```

Comment:
we should also add a note that for this to work, standby replicas must be turned on using `ksql.streams.num.standby.replicas >= 1` . Ideally we should auto set the standby replication.. Thats another way to go about it.. May be file an issue for later?